app/server/mergedServerMain.ts

```diff
@@ -104,6 +104,7 @@ export async function main(port: number, serverTypes: ServerType[],
   }
 
   server.addHealthCheck();
+  server.addBootPage();
```

Comment:
Does this need to be in a conditional (e.g. only for home servers)?

Reply:
Added a conditional for simplicity, but might actually be a useful page to have later on individual doc workers if routable.